src/components/Tag/index.tsx

```diff
@@ -98,17 +132,17 @@ export interface InternalTagProps {
 	/** The operator that goes between the name and value of the tag */
 	operator?: string;
 	/** An array of name-value pairs, with an optional delimiter to be used between the previous and current tag entry */
-	multiple?: Array<{
-		value?: string;
-		name?: string;
-		operator?: string;
-		prefix?: string;
-	}>;
+	multiple?: TagItem[];
 	/** Callback method, that if passed, a "close" button will be added to the right-hand side of the tag */
 	onClose?: (event: React.MouseEvent<HTMLElement, MouseEvent>) => void;
 	/** Callback method, that if passed, the tag will become clickable */
 	onClick?: (event: React.MouseEvent<HTMLElement, MouseEvent>) => void;
 	className?: string;
+	/** If set, limits the max number of items to display if multiple name-value pairs are specified.
```

Comment:
What I suggest is removing this for now, and just defaulting to 3 in all cases. If we really feel like we want it to be customizable, we can add it in the future, but I think it makes sense as a general rule to limit it to the same value, eg. 3

Reply:
This sounds good to me